src/main/scala/org/neo4j/spark/Neo4jOptions.scala

```diff
@@ -110,7 +144,7 @@ case class Neo4jDriverOptions(
                              ) extends Serializable {
 
   def toDriverConfig: Config = {
-    val builder = Config.builder()
+    val builder = Config.builder().withUserAgent("neo4j-spark-connector")
```

Comment:
include version please.  Inevitably we'll have a bug later and will need to know what a customer is using

Reply:
done